api/FetchEvent.json

```diff
@@ -299,6 +299,8 @@
       },
       "preloadResponse": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/PreloadResponse",
```

Comment:
The capitalization here is wrong, does it need to be renamed/redirected on MDN?

Reply:
Yes, I think so. I'll open a PR there (tomorrow). Once fixed I'll commit the requested change here.